src/libcrun/cgroup.c

```diff
@@ -1073,6 +1078,14 @@ int enter_systemd_cgroup_scope (runtime_spec_schema_config_linux_resources *reso
         }
     }
 
+  if (userdata.unit && userdata.result)
+    {
+      if (strcmp(userdata.result, "done") != 0)
```

Comment:
do we need the same check for `destroy_systemd_cgroup_scope` ?

Reply:
fixed